docs/low-code-spark/gems/source-target/warehouse/cosmos.md

```diff
@@ -0,0 +1,134 @@
+| Enable Write Bulk                  | `False`  | `true`                         | Cosmos DB Item Write bulk enabled                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+
+### Patch Specific Configurations
+
+These Configurations are valid only when `Write Strategy` is given as `ItemPatch`
+
+| Property                     | Required | Default Value | Description                                                                                                                                                                                                                                                                                                           |
+| ---------------------------- | -------- | ------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Default Patch Operation Type | `True`   | `Replace`     | Default Cosmos DB patch operation type. Supported ones include none, add, set, replace, remove, increment. Choose none for no-op, for others please reference here for full context.                                                                                                                                  |
+| Patch Column Configs         | `False`  | `None`        | Cosmos DB patch column configs. It can container multiple definitions matching the following patterns separated by comma. col(column).op(operationType) or col(column).path(patchInCosmosdb).op(operationType) - The difference of the second pattern is that it also allows you to define a different cosmosdb path. |
```

Comment:
"It can container multiple definitions matching the following patterns separated by comma" doesn't really make sense. Can we clarify this?